client/app/index.html

```diff
@@ -122,6 +122,7 @@
   <script src="src/directives/validAmount.directive.js"></script>
   <script src="src/directives/copyToClipboard.directive.js"></script>
   <script src="src/directives/backgroundStyle.directive.js"></script>
+  <script src="src/directives/open-external.directive.js"></script>
```

Comment:
The name is inconsistent with the others here

Reply:
I've changed the rest of the files to kebab-case.